zk/tx/tx_test.go

```diff
@@ -83,7 +86,7 @@ func TestDecodePost155Tx(t *testing.T) {
 	post155Bytes, err := hex.DecodeString(post155[2:])
 	require.NoError(t, err)
 	post155Bytes = append(post155Bytes, 75)
-	tx, pct, err := DecodeTx(post155Bytes, forkID5)
+	tx, pct, err := DecodeTx(post155Bytes, 0, forkID5)
```

Comment:
These can get rebased over by the test fixes branch where they are updated properly.